<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgGmicInputOutput</class>
 <widget class="QWidget" name="WdgGmicInputOutput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="KisFilterPreviewWidget" name="previewViewport">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="previewCheckBox">
     <property name="text">
      <string>Preview</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="zoomInButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Zoom In</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoomOutButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>Zoom Out</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="inputLayerLabel">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="inputCombo"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="outputLayerLabel">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QComboBox" name="outputCombo"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="outputPreviewLabel">
     <property name="text">
      <string>Preview mode</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="outputPreviewCombo"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="previewSizeLabel">
     <property name="text">
      <string>Preview size</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="previewSizeCombo"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisFilterPreviewWidget</class>
   <extends>QWidget</extends>
   <header>kis_filter_preview_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
